core/src/main/scala/filodb.core/query/TransientRow.scala

```diff
@@ -9,6 +10,21 @@ trait MutableRowReader extends RowReader {
   def setBlob(columnNo: Int, base: Array[Byte], offset: Int, length: Int): Unit
 }
 
+final class NaNRowReader(var timestamp: Long) extends RowReader {
+  override def isNaN: Boolean = true
+  override def notNull(columnNo: Int): Boolean = true
```

Comment:
Should this be false?

Reply:
column 0 not null? and the others are null?